pkg/driver/mocks/mock_cloud.go

```diff
@@ -1,243 +1,508 @@
 // Code generated by MockGen. DO NOT EDIT.
-// Source: github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud (interfaces: Cloud)
+// Source: pkg/cloud/cloud.go
 
 // Package mocks is a generated GoMock package.
 package mocks
 
 import (
 	context "context"
-	reflect "reflect"
-
+	aws "github.com/aws/aws-sdk-go/aws"
+	request "github.com/aws/aws-sdk-go/aws/request"
+	ec2 "github.com/aws/aws-sdk-go/service/ec2"
 	gomock "github.com/golang/mock/gomock"
 	cloud "github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud"
+	reflect "reflect"
 )
 
-// MockCloud is a mock of Cloud interface.
+// MockEC2 is a mock of EC2 interface
```

Comment:
how come it's mocking ec2 interface now? the ec2 mock should go to mock_ec2.go (looking at script https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/hack/update-gomock#L21)

Reply:
Interesting.. Can you try on your end? I just ran make bin/mockgen under project root path and get this EC2 mock added into the mock_cloud.go

Reply:
I tried with latest mockgen, it only changed my go.sum : /
```
~/g/s/g/k/aws-ebs-csi-driver (master|✔) $ rm bin/mockgen 
~/g/s/g/k/aws-ebs-csi-driver (master|✔) $ make mockgen
go get github.com/golang/mock/mockgen@latest
go: found github.com/golang/mock/mockgen in github.com/golang/mock v1.5.0
go: downloading golang.org/x/tools v0.0.0-20191119224855-298f0cb1881e
go: downloading golang.org/x/mod v0.3.0
./hack/update-gomock
```

Reply:
figured that out.. I didn't specify the interface name when I ran the command.. updated now